compiler/rustc_feature/src/unstable.rs

```diff
@@ -360,6 +360,8 @@ declare_features! (
     (unstable, associated_type_defaults, "1.2.0", Some(29661)),
     /// Allows `async || body` closures.
     (unstable, async_closure, "1.37.0", Some(62290)),
+    /// Allows implementing `AsyncDrop`.
+    (incomplete, async_drop, "CURRENT_RUSTC_VERSION", Some(126482)),
```

Comment:
I don't see feature gates in the compiler being tested in the other PRs. If this all hinges on the library feature gate, then let it hinge on the library feature gate, and remove the compiler gate.